public/stylesheets/style.css

```diff
@@ -123,6 +123,36 @@ div[class^="col-"] {
 .reviewAuthor {margin: 0 5px;}
 .reviewTime {color: #ccc;}
 
+/*CSS for custom footer ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+ul {
+  list-style-type: none;
+}
+
+a.bottomlink {
```

Comment:
Instead of adding your own custom styles here look into leveraging bootstraps classes. 
This will fix your responsive issues. 
Google bootstrap horizontal lists that might help